js/libs/templates.js

```
@@ -63,11 +117,71 @@
+    templateCache.set(organizationId, organizationTemplates);
+
+    return organizationTemplates;
+  });
+};
+
+/**
+ * Clear the template cache for the current organization
+ * Useful when templates are created/updated/deleted
+ */
+Fliplet.FormBuilder.clearTemplateCache = function() {
+  var organizationId = Fliplet.Env.get('organizationId');
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)